frontend/web/components/pages/AnimeDetails.js

```diff
@@ -28,25 +29,56 @@ class AnimeDetails extends React.Component {
   render () {
     const { animeCache } = this.props;
     let content = null;
-    let details = animeCache[this.props.params.id];
+    let details = animeCache[this.props.params.id]
     if (details) {
-      console.log('got details');
-      let averageScore = (Math.round(parseInt(details.average_score, 10)))/10;
-      let description = details.description.replace(/<[^>]*>/ig, "");
+      console.log('got details')
+      console.log(details);
+      let averageScore = (Math.round(parseInt(details.average_score, 10)))/10
```

Comment:
I'd abstract this out into another function. We're actually doing this a lot in different places, maybe we can abstract this into a utils file and import it, if we ever decide to do scores out of 5 or something we can just change the logic in one place